rbd.c

```diff
@@ -246,11 +246,66 @@ static int tcmu_rbd_service_status_update(struct tcmu_device *dev,
 #endif /* LIBRADOS_SUPPORTS_SERVICES */
 
 #if defined LIBRADOS_SUPPORTS_GETADDRS || defined RBD_LOCK_ACQUIRE_SUPPORT
-static void tcmu_rbd_rm_stale_entry_from_blacklist(struct tcmu_device *dev, char *addrs)
+
+static int tcmu_rbd_remove_blocklist(struct tcmu_device *dev, char *addr)
+{
+	struct tcmu_rbd_state *state = tcmur_dev_get_private(dev);
+	char *cmd;
+	int ret;
+
+	ret = asprintf(&cmd,
+		       "{\"prefix\": \"osd blocklist\","
+		       "\"blocklistop\": \"rm\","
+		       "\"addr\": \"%s\"}",
+		       addr);
+	if (ret < 0) {
+		tcmu_dev_warn(dev, "Could not allocate command. (Err %d)\n",
+			      ret);
+		return -1;
+	}
+	ret = rados_mon_command(state->cluster, (const char**)&cmd, 1, NULL, 0,
+				NULL, NULL, NULL, NULL);
+	free(cmd);
+	if (ret < 0) {
+		tcmu_dev_warn(dev, "Could not rm blocklist entry '%s'. (Err %d)\n",
```

Comment:
`tcmu_dev_warn` is probably too high -- on an old cluster, it would pollute the logs and make it appear that something is wrong.  I'd switch to `tcmu_dev_dbg` here.